src/handler.rs

```diff
@@ -54,7 +54,8 @@ impl DivviupApi {
                 logger(),
                 origin_router()
                     .with_handler(config.app_url.as_ref(), static_assets(&config))
-                    .with_handler(config.api_url.as_ref(), api(&db, &config)),
+                    .with_handler(config.api_url.as_ref(), api(&db, &config))
+                    .with_default_handler(api(&db, &config)),
```

Comment:
I wrote almost identical code as this and then realized that it's identical to:

```rust
origin_router().with_handler(config.app_url.as_ref(), static_assets(&config),
api(&db, &config)
```

since the origin_router.default_handler is the same as just falling through and not halting

Reply:
Indeed, this is much better!